### PR TITLE
opentelemetry-gke: add self observability dashboards for the OTel Collector

### PR DESCRIPTION
**Pull request by ridwanmsharif**

### CODE REVIEW

File: dashboards/opentelemetry-gke/metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+sample_dashboards:
+  -
+    category: OpenTelemetry Collector
+    id: opentelemetry-collector
+    display_name: OpenTelemetry Collector
```
[the remark this answers is not in the document]
Comment (ridwanmsharif) [reply dashboards/opentelemetry-gke/metadata.yaml]: Changed the display name in the dashboard

File: dashboards/opentelemetry-gke/opentelemetry-collector.json [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+                        "resource.label.\"job\"",
+                        "resource.label.\"instance\""
+                      ]
+                    }
+                  },
+                  "unitOverride": "",
+                  "outputFullDuration": false
+                },
+                "plotType": "LINE",
+                "legendTemplate": "${metric.labels.status}${resource.labels.instance}",
```
[the remark this answers is not in the document]
Comment (ridwanmsharif) [reply dashboards/opentelemetry-gke/opentelemetry-collector.json]: Done.

**Comment by ridwanmsharif** [comment]
I updated the dashboard so it doesn't refer to GKE specifically. The OTel collector can be run with the same self observability metrics on GCE as well.

File: dashboards/opentelemetry-collector/opentelemetry-collector.json [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+            }
+          }
+        }
+      },
+      {
+        "yPos": 16,
+        "width": 24,
+        "height": 16,
+        "widget": {
+          "title": "Cloud Observability API Request Count",
```
Comment (johnbryan) [inline dashboards/opentelemetry-collector/opentelemetry-collector.json:109]: Would this metric include RPCs to other observability backends too? If so could we give this chart a more generic name?

**Review by johnbryan — APPROVED** [review]
Looks good, just one comment on a chart title

File: dashboards/opentelemetry-collector/opentelemetry-collector.json (hunk 3 above)
Comment (ridwanmsharif) [reply dashboards/opentelemetry-collector/opentelemetry-collector.json:109]: This tracks RPCs only to GCP Cloud Observability

**Comment by ridwanmsharif** [comment]
Thanks for the review @johnbryan , I don't have write access but let me know if you want me to change anything else before you merge the PR

**Comment by damemi** [comment]
@ridwanmsharif could you add a link to this from the readme in https://github.com/GoogleCloudPlatform/otlp-k8s-ingest with a little section on using it with the self-observability metrics in the readme?

**Pull request merged**